README.md

```diff
@@ -8,12 +8,27 @@ to be added to Drupal core and installed using [Composer](https://getcomposer.or
 below use the [Apigee Devportal Kickstart composer project](https://github.com/apigee/devportal-kickstart-project-composer)
 to install Drupal core and this installation profile to create a site.
 
-# Prerequisites
+## Module status
+
+The core functionality of this module is in active development. The features and functionality will be changing as we
+continue work.
+
+We encourage you to download and evaluate the module, and to use our 
+[GitHub issue queue](https://github.com/apigee/apigee-devportal-kickstart-drupal/issues) to give feedback, ask questions, 
+or log issues.
+
+## Planned features
+* **Demonstration theme & content:** Demonstration content with the look and feel of a branded developer portal
+* **API Documentation:** Example of API documentation published on site using OpenAPI specifications
```

Comment:
I think this is meant to say that we're adding API documentation, yes? If so, perhaps this could be written as, "API documentation rendered from OpenAPI specifications."